dockerfiles/alpine_3.5

```diff
@@ -18,17 +18,17 @@ ENV TARANTOOL_VERSION=${TNT_VER} \
     GPERFTOOLS_REPO=https://github.com/gperftools/gperftools.git \
     GPERFTOOLS_TAG=gperftools-2.5 \
     LUAROCKS_URL=https://github.com/tarantool/luarocks/archive/6e6fe62d9409fe2103c0fd091cccb3da0451faf5.tar.gz \
-    LUAROCK_VSHARD_VERSION=0.1.14 \
-    LUAROCK_AVRO_SCHEMA_VERSION=3.0.3 \
-    LUAROCK_EXPERATIOND_VERSION=1.0.1 \
-    LUAROCK_QUEUE_VERSION=1.0.6 \
+    LUAROCK_VSHARD_VERSION=0.1.18 \
+    LUAROCK_AVRO_SCHEMA_VERSION=3.0.6 \
+    LUAROCK_EXPERATIOND_VERSION=1.1.1 \
+    LUAROCK_QUEUE_VERSION=1.1.0 \
     LUAROCK_CONNPOOL_VERSION=1.1.1 \
-    LUAROCK_HTTP_VERSION=1.1.0 \
+    LUAROCK_HTTP_VERSION=1.2.0 \
     LUAROCK_MEMCACHED_VERSION=1.0.1 \
-    LUAROCK_METRICS_VERSION=0.10.0 \
+    LUAROCK_METRICS_VERSION=0.12.0 \
     LUAROCK_TARANTOOL_PG_VERSION=2.0.2 \
     LUAROCK_TARANTOOL_MYSQL_VERSION=2.0.1 \
-    LUAROCK_TARANTOOL_MQTT_VERSION=1.2.1 \
+    LUAROCK_TARANTOOL_MQTT_VERSION=1.5.0 \
```

Comment:
AFAIU, we will not tag new images based on Apline 3.5.

(Sure, we should revisit the repository layout in a future. Possibly use some template engine.)

Reply:
Ok, removed changes for Alpine 3.5.